sample-apps/SampleAppSpec.md

```diff
@@ -227,6 +365,79 @@ These are Key Value pairs to be added on traces.
 }
 ```
 
+**Expected Traces Test Files**
+
+The subsegment span names will be tested against and are displayed below.
+
+The outgoing-sampleapp trace *will not be tested* for this iteration as it is out of scope due to no need to test this functionality.  However, if we were to test this sample app chaining then we would have the assertion validate shape of the trace and it's subsegments/spans.  For example:
+E.g. Go → Go or Go → Ruby or Ruby → Go
+
+
+`/outgoing-http-call`
```

Comment:
I have the same concern that this is leaking implementation details of the test suite into the spec document. 

Does the spec provide clear instructions on how traces must be built so that this data shape will be generated? 

If so then I think this and the below content is not required.

Reply:
I agree that this could be leaking implementations details so I will revise the spec to clearly state what will be tested for to match the trace map data